client/components/tinymce/plugins/simple-payments/style.scss

```diff
@@ -92,6 +92,69 @@
 			text-decoration: none;
 		}
 	}
+
+	.dialog__editor-simple-payments-modal-figure-container {
+		position:relative;
```

Comment:
Maybe add a prettier format to the top of this file?

Reply:
Updated in 2143e538ba4897342daf11d7d0240793291ae50d.